setup.py

```diff
@@ -15,7 +15,7 @@
     package_data={'bluesky': ['schema/*.json']},
     install_requires=['jsonschema', 'traitlets', 'cycler',
                       'numpy', 'matplotlib', 'super_state_machine',
-                      'historydict'],
+                      'historydict', 'event_model'],
```

Comment:
You guys are right. *If* we include `event-model` then it needs a hyphen (`-`).  Given that we do not have it as a pypi package, we should not include it in the install_requires section.  For example:

```
conda create -p /tmp/foo bluesky python=3.5
source activate bluesky
conda remove bluesky
cd /path/to/this/bluesky/branch
python setup.py develop
```
The above will barf because it cannot find event-model on pypi:

```
$ psd
running develop
running egg_info
writing bluesky.egg-info/PKG-INFO
writing requirements to bluesky.egg-info/requires.txt
writing top-level names to bluesky.egg-info/top_level.txt
writing dependency_links to bluesky.egg-info/dependency_links.txt
reading manifest file 'bluesky.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'bluesky.egg-info/SOURCES.txt'
running build_ext
Creating /tmp/foo/lib/python3.5/site-packages/bluesky.egg-link (link to .)
bluesky 0.4.3+87.gd35b09f.dirty is already the active version in easy-install.pth

Installed /home/edill/dev/python/bluesky
Processing dependencies for bluesky==0.4.3+87.gd35b09f.dirty
Searching for event-model
Reading https://pypi.python.org/simple/event-model/
Couldn't find index page for 'event-model' (maybe misspelled?)
Scanning index of all packages (this may take a while)
Reading https://pypi.python.org/simple/
No local packages or download links found for event-model
error: Could not find suitable distribution for Requirement.parse('event-model')
```

Therefore I would strongly suggest that we do not include this package in the setup.py.  We *could* also publish it to pypi, but then we have issues connecting to pypi from behind the NSLS-II firewall.  Overall, I think this causes more trouble than it solves.